global.json

```
@@ -0,0 +1,3 @@
+{
+  "sdk": { "version": "6.0.100" }
```

This should make it work with the latest .NET 6 SDK:

```suggestion
  "sdk": { "version": "6.0.100" },
  "rollForward": "latestMinor"
```